packet/packet.go

```diff
@@ -33,38 +35,61 @@ func Unmarshal(b []byte) (p ArtNetPacket, err error) {
 	case code.OpAddress:
 		p = &ArtAddressPacket{}
 	case code.OpInput:
+		return nil, notImplErr
```

Comment:
I would suggest merging all of the `return nil, notImplErr` cases in one case using `case a, b: ` notation.